decidim-conferences/spec/controllers/admin/conferences_controller_spec.rb

```diff
@@ -42,7 +42,8 @@ module Conferences
           expect(Decidim::Conferences::Admin::ConferenceForm).to receive(:from_params).with(hash_including(id: conference.id.to_s)).and_call_original
           patch :update, params: { slug: conference.id, conference: conference_params }
 
-          expect(response).to be_successful
```

Comment:
I see that you've fixed a failing spec with this change. What I don't understand is why this was happening in this PR as it doesn't seem related (or I can't seem the relation at least).

Is it just a flaky spec? Do you have any theory about this?

Reply:
The old spec is wrong after the fix is applied. It's related to this change:
https://github.com/decidim/decidim/blob/3c48d3a46bea60b1071abd42f70c61e68d406dfe/decidim-core/lib/decidim/attributes/localized_date.rb#L19

In this spec we are setting `Date` values to the start and end date which were converted to `nil` in the previous version -> This causes an error on the form, similar if you would actually submit the form with empty start and end dates (you have to remove the `required` attribute from the start and end date inputs to be able to submit the form):
![Form error](https://user-images.githubusercontent.com/864340/191734790-bb3863f4-10f1-4bc7-8fc2-6492eb4d110c.png)

So what happened before was that this was testing that sending the response is HTTP 200, which it is in this case as it will just directly render the form with the error message.

After this fix, the start and end dates are assigned correctly which causes the response to be a redirect (HTTP 302) instead of a "success" (HTTP 200).

So the test is not testing what it's supposed to test. After this fix the test is also correct.

Reply:
Ah, ok, makes perfect sense!! Thanks for the explanation 👍🏽